app/src/main/java/com/example/campusguide/directions/Route.kt

```diff
@@ -47,7 +91,19 @@ class Route constructor(private val map: GoogleMap, private val activity: AppCom
                  * addPolyline throws an exception if it is not run on the Ui thread.
                  */
                 activity.runOnUiThread {
-                    polyline = map.addPolyline(PolylineOptions().addAll(decodedAsGoodLatLng))
+                    val polyOptions = PolylineOptions()
+                        .color(COLOR_BLUE_ARGB.toInt())
+                        .pattern(PATTERN_POLYGON_ALPHA)
+                        .addAll(decodedAsGoodLatLng)
+                    polyline = map.addPolyline(polyOptions)
+                    begin = map.addMarker(startPoint)
+                    dest = map.addMarker(endPoint)
+                    map.moveCamera(
+                        CameraUpdateFactory.newLatLngZoom(
+                            routeBounds.center,
+                            Constants.ZOOM_STREET_LVL
```

Comment:
This won't work when the route covers a large area: the zoom should be calculated based on the route bounds so that the whole route can be seen at once.